app/src/components/toggleButtonGroup/ToggleButtonGroup.tsx

```diff
@@ -73,12 +58,14 @@ export const ToggleButtonGroup = ({
   ...props
 }: ToggleButtonGroupProps & SizingProps & StylableProps) => {
   return (
-    <AriaToggleButtonGroup
-      data-size={size}
-      className={classNames("ac-toggle-button-group", className)}
-      css={css(baseToggleButtonGroupCSS, cssProp)}
-      selectionMode={selectionMode}
-      {...props}
-    />
+    <SizeProvider size={size}>
```

Comment:
propagate size to the children via context.

Reply:
I'm a big fan of this pattern. React aria uses it very well